src/plugins/unified_histogram/public/chart/get_lens_attributes.ts

```diff
@@ -0,0 +1,174 @@
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import type { DataView, DataViewField } from '@kbn/data-views-plugin/public';
+import type { AggregateQuery, Filter, Query } from '@kbn/es-query';
+import { i18n } from '@kbn/i18n';
+import type {
+  CountIndexPatternColumn,
+  DateHistogramIndexPatternColumn,
+  GenericIndexPatternColumn,
+  TermsIndexPatternColumn,
+  TypedLensByValueInput,
+} from '@kbn/lens-plugin/public';
+import { fieldSupportsBreakdown } from './field_supports_breakdown';
+
+export const getLensAttributes = ({
+  filters,
+  query,
+  dataView,
+  timeInterval,
+  breakdownField,
+}: {
+  filters: Filter[];
+  query: Query | AggregateQuery;
+  dataView: DataView;
+  timeInterval: string | undefined;
+  breakdownField: DataViewField | undefined;
+}) => {
+  const showBreakdown = breakdownField && fieldSupportsBreakdown(breakdownField);
+
+  let columnOrder = ['date_column', 'count_column'];
+
+  if (showBreakdown) {
+    columnOrder = ['breakdown_column', ...columnOrder];
+  }
+
+  let columns: Record<string, GenericIndexPatternColumn> = {
+    date_column: {
+      dataType: 'date',
+      isBucketed: true,
+      label: dataView.timeFieldName ?? '',
+      operationType: 'date_histogram',
+      scale: 'interval',
+      sourceField: dataView.timeFieldName,
+      params: {
+        interval: timeInterval ?? 'auto',
+      },
+    } as DateHistogramIndexPatternColumn,
+    count_column: {
+      dataType: 'number',
+      isBucketed: false,
+      label: i18n.translate('unifiedHistogram.countColumnLabel', {
+        defaultMessage: 'Count of records',
+      }),
+      operationType: 'count',
+      scale: 'ratio',
+      sourceField: '___records___',
+      params: {
+        format: {
+          id: 'number',
+          params: {
+            decimals: 0,
+          },
+        },
+      },
+    } as CountIndexPatternColumn,
+  };
+
+  if (showBreakdown) {
+    columns = {
+      ...columns,
+      breakdown_column: {
+        dataType: 'string',
+        isBucketed: true,
+        label: i18n.translate('unifiedHistogram.breakdownColumnLabel', {
+          defaultMessage: 'Top 3 values of {fieldName}',
+          values: { fieldName: breakdownField?.displayName },
+        }),
+        operationType: 'terms',
+        scale: 'ordinal',
+        sourceField: breakdownField.name,
+        params: {
+          size: 3,
+          orderBy: {
+            type: 'column',
+            columnId: 'count_column',
+          },
+          orderDirection: 'desc',
+          otherBucket: true,
+          missingBucket: false,
+          parentFormat: {
+            id: 'terms',
+          },
+        },
+      } as TermsIndexPatternColumn,
+    };
+  }
+
+  return {
+    title: '',
```

Comment:
By adding the title like that it results on an empty breadcrumb that looks like that:
<img width="509" alt="image" src="https://user-images.githubusercontent.com/17003240/204280094-c9927ed3-433d-4676-8076-c1d15e653207.png">

I suggest to give the "Edit visualization" title here.

Reply:
Just saw that @jughosta also mentioned it above :)

Reply:
Done, thanks!
<img width="543" alt="Screen Shot 2022-11-28 at 6 06 53 PM" src="https://user-images.githubusercontent.com/25592674/204391024-b375c33f-9217-4967-a42d-5cde9d683567.png">